app/uk/gov/hmrc/agentfirelationship/controllers/RelationshipController.scala

```diff
@@ -48,11 +48,11 @@ class RelationshipController @Inject()(authAuditConnector: AuthAuditConnector,
 
   def findRelationship(arn: String, service: String, clientId: String): Action[AnyContent] = Action.async { implicit request =>
     implicit val auditData = new AuditData()
-    mongoService.findRelationships(arn, service, clientId) flatMap { result =>
+    mongoService.findRelationships(arn, service, clientId, RelationshipStatus.Active) flatMap { result =>
       if (result.nonEmpty) {
         Future.successful(Ok(toJson(result)))
       } else {
-        mongoService.findCeasedRelationships(arn, service, clientId) flatMap { cesaRelationship =>
+        mongoService.findCESARelationships(arn, service, clientId, RelationshipStatus.Active) flatMap { cesaRelationship =>
```

Comment:
The purpose of this check was to look at ceased (inactive) relathionships and if any exist then skip copying from CESA, how is it handled now?